### ISSUE TITLE
Booking Overview - Delete Booking Function

### ISSUE THREAD

**Issue by qy21gafy** [qy21gafy/AMOSSharedDeskFeatureBoard#16]
## User story
1. As a user 
2. I want / need to be able to cancel my bookings if my plans change
3. So that someone else can book that desk

## Acceptance criteria
* Delete function should be accessible from the booking overview page
* Before deleting the booking an "are you sure?" message should appear 
* Delete function should delete the booking information provided by the user for the ressource from the database
* After deletion the ressource should be free to be booked again 
* After deletion the ressource should not be visible in the booking overview page of the user anymore

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
